src/components/Book.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import classes from './Book.css';
```

Comment:
I am confused what this is for. And why are we importing an object (which is an **empty object**) in a components file from a css file?